wework/src/i18n/locales/en/common.json

```
@@ -61,11 +61,16 @@
     "archive_chats": "Archive chats",
     "archive_chat": "Archive chat",
     "archive_runtime_task": "Archive task",
+    "archive_runtime_task_pending": ", archiving soon",
+    "archive_runtime_task_undo": "Undo",
+    "archive_runtime_task_notice_close": "Dismiss archive notice",
     "runtime_task_running": "Running",
     "mark_runtime_task": "Mark task",
     "unmark_runtime_task": "Unmark task",
     "rename_project": "Rename project",
```

Biome (2.5.0)

[error] The key rename_project was already declared.

(lint/suspicious/noDuplicateObjectKeys)